api/hooks/delete_uninstall.js

```diff
@@ -16,7 +16,7 @@ module.exports = server => ({
     logger.info('Starting uninstall...');
 
     try {
-      await uninstall(req.pre.auth0.rules);
+      await uninstall({ client: req.pre.auth0 });
```

Comment:
Previously we only needed access to the [RulesManager](https://auth0.github.io/node-auth0/classes/management.RulesManager.html). I changed this to pass in the entire [ManagementClient](https://auth0.github.io/node-auth0/classes/management.ManagementClient.html) instead so that we can set the rules configs.